File: src/main/scala/org/joshjoyce/lumivore/db/SetupDatabase.scala
package org.joshjoyce.lumivore.db

object SetupDatabase {
  def main(args: Array[String]) {
    val db = new SqliteDatabase
    db.connect()
    db.createIndexes()
    //db.createTables()
    //Set("jpg", "dng", "rw2", "orf").foreach(db.addExtension)
    println("ok")
  }
}

File: src/main/java/org/joshjoyce/lumivore/db/SetupDatabase.java
package org.joshjoyce.lumivore.db;

public class SetupDatabase {
    public static void main(String[] args) {
        var db = new SqliteDatabase();
        db.connect();
        db.createTables();
        db.createIndexes();
        //Set("jpg", "dng", "rw2", "orf").foreach(db.addExtension)
        System.out.println("ok");
        System.out.println("foo");
    }
}
